objectivec/GPBMessage_PackagePrivate.h

```diff
@@ -42,6 +42,103 @@
 
 #import "GPBBootstrap.h"
 
+#if defined(__LP64__) && __LP64__
+#define GPB_ASM_PTR " .quad "
+#define GPB_ASM_PTRSIZE " 8 "
+#define GTP_ASM_ONLY_LP64(x) x
+#define GTP_ALIGN_SIZE " 3 "
+#else  // __LP64__
+#define GPB_ASM_PTR " .long "
+#define GPB_ASM_PTRSIZE " 4 "
+#define GTP_ASM_ONLY_LP64(x)
+#define GTP_ALIGN_SIZE " 2 "
+#endif  // __LP64__
+
+#define GPB_MESSAGE_CLASS_NAME_RAW GPBMessage
+#define GPB_STRINGIFY_INTERNAL(x) #x
```

Comment:
Don't we have two stringify defines already at the top of this file?  Can we use those or do we need unique ones again?

Reply:
Not at the top of this file, but I found them in GPBUtilities. Using those now.